lib/Retcon/Network/Server.hs

```
@@ -102,13 +106,17 @@ spawnServer cfg n = do
         sock   <- socket ctx Rep
         bind sock zmq_conn
         db     <- initBackend (PGOpts pg_conn)
-        return $  ServerState ctx sock cfg db
+        -- Setup ekg
+        ekgStore  <- initialiseMeters cfg
+        ekgServer <- Ekg.forkServerWith ekgStore "localhost" 8888
+        return $  ServerState ctx sock cfg db ekgServer
 
     stop :: ServerState -> IO ()
     stop state = do
         closeBackend $ state ^. serverStore
         close        $ state ^. serverSocket
         term         $ state ^. serverContext
+        killThread   $ Ekg.serverThreadId $ state ^. serverEkgServer
```

ok then, guess async exceptions are masked ¯\_(ツ)_/¯